javacutil/src/org/checkerframework/javacutil/InternalUtils.java

```diff
@@ -349,7 +349,9 @@ public static TypeMirror greatestLowerBound(
         if (t2.getKind() == TypeKind.WILDCARD) {
             return t1;
         }
-        return types.glb(t1, t2);
+        // call glb on unannotatedType, otherwise glb may cause a stack overflow error
```

Comment:
I would remove the reference to the issue here and explain in which cases we reach this point.